### PR TITLE
Jsonnet config simplify

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

File: pystarport/tests/test_expansion/utils.jsonnet [hunk 1]
```diff
@@ -4,9 +4,6 @@
     staked: '1000000000000000000stake',
     mnemonic: mnemonic,
   },
-  account(name, coins, mnemonic):: {
-    name: name,
-    coins: coins,
-    mnemonic: mnemonic,
-  },
+  validators(mnemonics):: [self.validator(mnemonic) for mnemonic in mnemonics],
+  accounts(objs):: [obj for obj in objs],
```
Comment (yihuang) [inline pystarport/tests/test_expansion/utils.jsonnet:8]: I don't like these two functions, it makes the config harder to customize.

**Review by yihuang — CHANGES_REQUESTED** [review]
Can you remove the `validators` and `accounts` util functions, they make the config harder to customize.

File: pystarport/tests/test_expansion/utils.jsonnet (hunk 1 above)
Comment (mmsqe) [reply pystarport/tests/test_expansion/utils.jsonnet:8]: so can rm this utils file?

File: flake.nix [hunk 2]
```diff
@@ -2,40 +2,16 @@
...
-            overrides = pkgs.poetry2nix.overrides.withDefaults
-              (self: super: {
-                pyparsing = super.pyparsing.overridePythonAttrs (
-                  old: {
-                    nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ self.flit-core ];
-                  }
-                );
-                jsonschema = super.jsonschema.overridePythonAttrs (
-                  old: {
-                    nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ self.hatchling self.hatch-vcs ];
```
Comment (yihuang) [inline flake.nix:34]: I think we do need these overrides to make the nix flake work: https://github.com/crypto-com/pystarport/runs/7124412665?check_suite_focus=true

File: flake.nix [hunk 3]
```diff
@@ -2,40 +2,16 @@
...
-                  }
-                );
-              });
-          };
+        packages.default = pkgs.poetry2nix.mkPoetryApplication {
+          projectDir = ./.;
+        };
```
Comment (yihuang) [inline flake.nix:14]:
```suggestion
        packages.default = pkgs.poetry2nix.mkPoetryApplication
          {
            projectDir = ./.;
            overrides = pkgs.poetry2nix.overrides.withDefaults
              (self: super: {
                pyparsing = super.pyparsing.overridePythonAttrs (
                  old: {
                    nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ self.flit-core ];
                  }
                );
                jsonschema = super.jsonschema.overridePythonAttrs (
                  old: {
                    nativeBuildInputs = (old.nativeBuildInputs or [ ]) ++ [ self.hatchling self.hatch-vcs ];
Member
@[yihuang](https://github.com/yihuang) yihuang [now](https://github.com/crypto-com/pystarport/pull/57/files#r910581326)
I think we do need these overrides to make the nix flake work: https://github.com/crypto-com/pystarport/runs/7124412665?check_suite_focus=true

@yihuang	Reply...
                  }
                );
              });
          };
```

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**